### DISCUSSION TITLE
2022-07-18 PyAerocom Meeting

### DISCUSSION THREAD

**Discussion by avaldebe** [metno/pyaerocom#706] [Development meetings]
## Last meeting

July 11th #698

## PRs and Issues

- #703
   - template for new readers, e.g
   - #543, and
   - #603
   - requires change of packaging from `setuptools` to `flit` foe editable installs during development
- #702
   - deals with #692
   - waiting review
- #696 
   - scope?
   - could include #605?
- #691 
   - deals with #604
   - 90% done (no change since last week)
- #664
   - deals with #663
   - status?
- #694
  - deals with #692
  - is more needed?

## Other issues

- #704
- #701
- #699 
- #678: waiting for feedback
- #642: status?
- #625: waiting for feedback
- #539: scheduled for next release, is it feasible?

**Comment by lewisblake** [comment]
Filtering nonphysical values for EBAS is a bit more challenging than the scope of my PR #696. We will need to address https://github.com/metno/pyaerocom/issues/708 as it's own issue, making new methods for EBAS which filters obs on species-specific criteria or potentially a new class
